src/pages/Test/index.js

```diff
@@ -0,0 +1,154 @@
+    _id: testId,
+    content: "Test",
+    timeLimit: 60,
+    questions: []
+  });
+
+  const [results, setResults] = useState({});
+  const [checked, setChecked] = useState({});
+
+  const getTestDetail = () => {
+    getTestDetailApi(testId).then((data) => {
+      if (data.error) {
+        toast.error(data.message);
+      } else {
+        const test = data?.data;
+        setTestDetail(test);
+
+        const initResults = test.questions.map((question) => ({
+          questionId: question._id,
+          answers: []
+        }));
+
+        setChecked({ ...Array(test.questions.length).fill([]) });
+        setResults({ ...initResults });
+      }
+    });
+  };
+
+  const handleChecked = (id, isMultiChoice, index) => {
+    let newChecked;
+
+    if (isMultiChoice) {
+      const isChecked = checked[index].includes(id);
+
+      newChecked = isChecked
+        ? checked[index].filter((item) => item !== id)
+        : [...checked[index], id];
+    } else {
+      newChecked = [id];
+    }
+
+    const resultChange = { ...results[index], answers: newChecked };
+    setResults({
+      ...results,
+      [index]: resultChange
+    });
+
+    setChecked({ ...checked, [index]: newChecked });
+  };
+
+  const handleSubmitTest = () => {
+    const userId = JSON.parse(localStorage.getItem("userId"));
+    const newTestResult = {
+      testId: testId,
+      userId: userId,
+      results: Object.values(results)
+    };
+
+    submitTestApi(newTestResult)
+      .then((data) => {
+        if (data.error) toast.error(data.message);
+        else toast.success("Submit test success");
+      })
+      .catch(() => toast.error("Submit test failed"));
+  };
+
+  const defaultRenderer = ({ hours, minutes, seconds, completed }) => {
+    if (completed) {
+      // Show test result
+    } else
+      return (
+        <span>
+          {minutes}:{seconds}
+        </span>
+      );
+  };
+
+  useEffect(() => {
+    getTestDetail();
+  }, []);
+
+  return (
+    <div className={styles.home}>
+      <section className={`container ${styles.homeSlider}`}>
+        <h2>Test</h2>
+        <div className={styles.countdown}>
+          <Countdown
+            renderer={defaultRenderer}
+            date={startDate.current + testDetail.timeLimit * 60 * 1000}
+            key={startDate.current + testDetail.timeLimit * 60 * 1000}
+          />
+        </div>
+        <div className={styles.row}>
+          {testDetail &&
+            testDetail.questions?.map((question, index) => (
+              <div className={styles.question_form}>
+                <div className={styles.question}>
+                  <span className={styles.content}>
+                    Question {index}. {question.content}
+                  </span>
+                  <span className={styles.score}>Score: {question.score}</span>
+                </div>
+                <div className={styles.answers}>
+                  {question.answers.map((answer) => (
+                    <>
```

Comment:
it's fine to remove <> and </>

Reply:
OK. We used to Fragment instead. But it's a forgot moment. 😅